docs/authorization/globus_app/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _globus_app:
+
+Globus App
+==========
+
+For a narrative-style introduction to the concepts contained within GlobusApp, see
+:ref:`using_globus_app`.
```

Comment:
:rocket: 

Great choice to split up the doc into reference vs narrative at this point in the tree!